### PR TITLE
FAQs page redesign and  Header component changed from class based component to  functional component

### PR DESCRIPTION
**Pull request by Ed-Gah**
## File updated
`componets/Header.js`
`utils/constants.js`

- Restored `pages/api/auth/[...nextauth].js`

### CODE REVIEW

**Comment by coderwassananmol** [comment]
@Ed-Gah While clicking on the username on the right, Logout option is not coming. Please fix it.

File: utils/constants.js [hunk 1]
```diff
@@ -29,16 +29,23 @@ export const faq_data = [
...
-      "Archives are supposed to be read-only and deleting is a 'write' operation. So it doesn't support.",
+      "Archives are supposed to be read-only and deleting is a 'write' operation. So it is not supported.",
   },
 ];
+
+export const header_links = [
+  { name: "Upload", href: "/", id: "form-tabs-1" },
+  { name: "Queue", href: "/queue", id: "form-tabs-2" },
+  { name: "Stats", href: "/stats", id: "form-tabs-3" },
+  { name: "FAQs", href: "/faqs", id: "form-tabs-4" },
```
Comment (coderwassananmol) [inline utils/constants.js:50]: Remove "," from here.

File: utils/constants.js [hunk 2]
```diff
@@ -29,16 +29,23 @@ export const faq_data = [
...
     ans:
-      "If an appropriate license is attached to the book which allows it to be archived(which doesn't allow copyright infringement suits, etc.), then book can be uploaded.",
+      "If an appropriate license is attached to the book which allows it to be archived (which doesn't allow copyright infringement suits, etc.), then book can be uploaded.",
   },
   {
     que: "Can I delete or undo an upload?",
     ans:
-      "Archives are supposed to be read-only and deleting is a 'write' operation. So it doesn't support.",
+      "Archives are supposed to be read-only and deleting is a 'write' operation. So it is not supported.",
   },
```
Comment (coderwassananmol) [inline utils/constants.js:43]: Remove "," from here.

File: pages/api/auth/[...nextauth].js [hunk 3]
```diff
@@ -7,8 +7,8 @@ export const authOptions = {
   providers: [
     WikimediaProvider({
       clientId: process.env.WIKIMEDIA_CLIENT_ID,
-      clientSecret: process.env.WIKIMEDIA_CLIENT_SECRET
-    })
+      clientSecret: process.env.WIKIMEDIA_CLIENT_SECRET,
+    }),
```
Comment (coderwassananmol) [inline pages/api/auth/[...nextauth].js:11]: Please remove "," from here.

**Review by coderwassananmol — CHANGES_REQUESTED** [review]

**Comment by Ed-Gah** [comment]
> @Ed-Gah While clicking on the username on the right, Logout option is not coming. Please fix it.

Okay Sir

File: utils/constants.js (hunk 1 above)
Comment (Ed-Gah) [reply utils/constants.js:50]: Sir each of this represents an Item in an array. If I remove this commas I will cause errors.
Screenshot from 2023-10-10 21-38-33

**Comment by Ed-Gah** [comment]
Done Sir

Screenshot from 2023-10-10 21-42-01

**Comment by Parthiv-M** [comment]
Hello, if you are concentrating on the FAQs page (along with the header) in this PR, could you please rename the PR accordingly, instead of the current title to avoid confusion?

**Comment by Ed-Gah** [comment]
I will do that Sir @Parthiv-M

**Comment by Ed-Gah** [comment]
Sir @Parthiv-M I have done the changes accordingly. 
I have also removed the white space from the Footer based on Sir @coderwassananmol comments on the other PR.
Screenshot from 2023-10-13 03-18-39

**Comment by coderwassananmol** [comment]
Closing this PR since #209 this is already present.

**Pull request closed without merging**